frontend/src/utils/formUtils.ts

```diff
@@ -7,3 +7,23 @@
 export const withNameSpace: Function = (nameSpace?: string, name?: string, index?: number) => {
   return [nameSpace ?? '', `${index ?? ''}`, name].filter(x => x).join('.');
 };
+
+/**
+ * The phoneFormatter is used to format the specified phone number value
+ * @param {string} phoneNumber This is the target phone number to be formatted
+ */
+export const phoneFormatter = (phoneNumber?: string) => {
+  if (!!phoneNumber) {
+    let result = phoneNumber;
+    const regex =
+      phoneNumber.length === 10
+        ? /(\d\d\d)[\s-]?(\d\d\d)[\s-]?(\d\d\d\d)/
+        : /\d(\d\d\d)[\s-]?(\d\d\d)[\s-]?(\d\d\d\d)/;
```

Comment:
If we are going to support country codes in the phone number, this regex only allows for single digit country codes., e.g. USA/CAN (1). Other countries, say UK (44) would not be matched here.

Not sure it's worth supporting phone numbers outside of CAN but worth mentioning here.

https://countrycode.org/